tests/framework/installer/ceph_installer.go

```diff
@@ -145,6 +146,13 @@ func (h *CephInstaller) CreateCephOperator() (err error) {
 		return errors.Errorf("Failed to create rook-operator pod: %v", err)
 	}
 
+	manifest := h.settings.readManifest("csi-operator.yaml")
+	manifest = strings.ReplaceAll(manifest, `namespace: rook-ceph`, `namespace: `+h.settings.OperatorNamespace)
```

Comment:
Instead of directly installing this in the rook-ceph namespace, we need to do it in the namespace specific for the test. But instead of just fixing that bug, let's pull in all the changes from #14766 that install/uninstall the CSI operator. Then we can begin a more complete transition to the csi operator in a future PR.

Reply:
done